superset/common/query_context_processor.py

```diff
@@ -74,6 +74,19 @@
 stats_logger: BaseStatsLogger = config["STATS_LOGGER"]
 logger = logging.getLogger(__name__)
 
+AGGREGATED_JOIN_COLUMN = "__aggregated_join_column"
+
+AGGREGATED_JOIN_GRAINS = {
```

Comment:
@michael-s-molina how are other time grains—day, hour, minute, etc.—handled?

Reply:
These time grains don't require an aggregated join column because time shifts don't influence the join criteria.